test/test_retrieve_dns.py

```diff
@@ -22,7 +22,6 @@ def test_verify_valid_dns(self):
 
     def test_verify_invalid_dns(self):
         invalid_dns = ("# Some comment",
-                       "/C=UK, /O=eScience, /OU=Lanchester, /L=Physics",
```

Comment:
As with that cert DN we were discussing today, it would be good if this format of DN was still rejected.

Reply:
or that no components contain trailing whitespace.

Reply:
So you mean `/C=UK, /O=eScience, /OU=Lanchester, /L=Physics` should just be interpreted as `C=UK, `, `O=eScience, `, etc. after splitting on `/`?

While they might be valid, I'd not want to accept those without good reason as they suggest something at least a bit off with the cert. Filtering for `, ` (comma space, 1st suggestion) doesn't work as that's valid in the middle of the field, but filtering for trailing whitespace in a field should work (2nd suggestion).

Reply:
Sorry I meant to say: no components end with the string ", " 

That should treat the more sensible use of "/OU=something, something" as valid, but not the less sensible "/OU=something, "